fdbclient/Schemas.cpp

```diff
@@ -482,6 +482,8 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
 )statusSchema"
                                                           R"statusSchema(
       "recovery_state":{
+         "time_since_last_db_turned_available_seconds":1,
+         "number_of_old_generations_of_tlogs":1,
```

Comment:
My vote is to leave this out if it gives the same info as `active_generations` (or, if we want to replace `active_generations`, then we should deprecate it and plan to remove it later).

Reply:
But `active_generations` is only logged at the beginning and end of recovery, relying on that might give us a wrong number. But even it's a wrong number, it can off the real count by at most 1, so I guess we can use it. I am going to remove `number_of_old_generations_of_tlogs`.

Reply:
Done